src/api/app/controllers/webui/feeds_controller.rb

```diff
@@ -15,12 +15,13 @@ def latest_updates
   def commits
     authorize @project, :source_access?
 
-    @start = params[:starting_at].present? ? starting_at(params[:starting_at]) : 7.days.ago
-    @finish = params[:ending_at].present? ? ending_at(params[:ending_at]) : nil
+    start = params[:starting_at].present? ? starting_at(params[:starting_at]) : 7.days.ago
```

Comment:
maybe would be simple with a `starting_at(params.fetch(:starting_at, 7.days.ago.to_s))` ?

Reply:
`before_action` to the rescue...